packages/components/alert/src/use-alert.ts

```diff
@@ -0,0 +1,123 @@
+import {HTMLNextUIProps, mapPropsVariants, PropGetter} from "@nextui-org/system";
+import {AlertSlots, SlotsToClasses} from "@nextui-org/theme";
+import {ReactRef, useDOMRef} from "@nextui-org/react-utils";
+import {AlertVariantProps} from "@nextui-org/theme/src/components/alert";
+import {ReactNode, useCallback, useMemo} from "react";
+import {alert} from "@nextui-org/theme";
+import {useState} from "react";
+import {objectToDeps} from "@nextui-org/shared-utils";
+
+interface Props extends HTMLNextUIProps<"div"> {
+  /**
+   * Ref to the DOM node.
+   */
+  ref?: ReactRef<HTMLDivElement | null>;
+
+  /**
+   * title of the alert message
+   */
+  title: string;
+
+  /**
+   * Main body of the alert message
+   */
+  description: ReactNode;
```

Comment:
i think `title` should be optional while `description` is mandatory.

Reply:
ok. will make the changes.